test/unit/mem/view/src/ViewPlainPtrTest.cpp

```diff
@@ -0,0 +1,228 @@
+/**
+ * \file
+ * Copyright 2018 Benjamin Worpitz
+ *
+ * This file is part of alpaka.
+ *
+ * alpaka is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * alpaka is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with alpaka.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// \Hack: Boost.MPL defines BOOST_MPL_CFG_GPU_ENABLED to __host__ __device__ if nvcc is used.
+// BOOST_AUTO_TEST_CASE_TEMPLATE and its internals are not GPU enabled but is using boost::mpl::for_each internally.
+// For each template parameter this leads to:
+// /home/travis/build/boost/boost/mpl/for_each.hpp(78): warning: calling a __host__ function from a __host__ __device__ function is not allowed
+// because boost::mpl::for_each has the BOOST_MPL_CFG_GPU_ENABLED attribute but the test internals are pure host methods.
+// Because we do not use MPL within GPU code here, we can disable the MPL GPU support.
+#define BOOST_MPL_CFG_GPU_ENABLED
+
+#include <alpaka/alpaka.hpp>
+#include <alpaka/test/acc/Acc.hpp>
+#include <alpaka/test/queue/Queue.hpp>
+#include <alpaka/test/mem/view/ViewTest.hpp>
+
+#include <alpaka/core/BoostPredef.hpp>
+#if BOOST_COMP_CLANG
+    #pragma clang diagnostic push
+    #pragma clang diagnostic ignored "-Wunused-parameter"
+#endif
+#include <boost/test/unit_test.hpp>
+#if BOOST_COMP_CLANG
+    #pragma clang diagnostic pop
+#endif
+
+#include <type_traits>
+#include <numeric>
+
+
+#if BOOST_COMP_GNUC
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wcast-align" // "cast from 'std::uint8_t*' to 'Elem*' increases required alignment of target type"
+#endif
+
+namespace alpaka
+{
+namespace test
+{
+namespace mem
+{
+namespace view
+{
+    //#############################################################################
+    //! 1D: sizeof(TIdx) * (5)
+    //! 2D: sizeof(TIdx) * (5, 4)
+    //! 3D: sizeof(TIdx) * (5, 4, 3)
+    //! 4D: sizeof(TIdx) * (5, 4, 3, 2)
+    // We have to be careful with the extents used.
+    // When Idx is a 8 bit signed integer and Dim is 4, the extent is extremely limited.
```

Comment:
it's `size_t` now (and the name is `Tidx` not `Idx`?)

Reply:
Here as well...